src/odemis/gui/util/__init__.py

```diff
@@ -282,3 +283,17 @@ class AttrDict(dict):
     def __init__(self, *args, **kwargs):
         super(AttrDict, self).__init__(*args, **kwargs)
         self.__dict__ = self
+
+
+def enable_cryo_tab_on_stage_position(tab, stage, pos):
+    """
+    Enable the given tab if the stage is in imaging position, disable it otherwise
+    :param tab:
+    :param stage:
+    :param pos:
+    :return:
+    """
+    if getCurrentPositionLabel(pos, stage) == IMAGING:
+        tab.panel.Enable()
+    else:
+        tab.panel.Disable()
```

Comment:
I'm getting used to creating helper functions to be used by different tabs. Let me know if that's the right place..

Also I saw your suggestion on Jira to disable the streams instead of the whole tab, what is the right way to do that? Thanks

Reply:
Let's not take care of the streams for now. It's a lot more work, and only useful for the localization tab. For the alignment tab, it should just be disabled, unless the stage is at IMAGING.